script/dump_measurements

```diff
@@ -1,12 +1,36 @@
-#!/bin/sh
-psql -U safecast --host $POSTGRESQL_ADDRESS safecast -c "\copy (select captured_at, ST_Y(location::geometry), ST_X(location::geometry), value, unit, location_name, device_id, md5sum, height, surface, radiation, created_at, measurement_import_id from measurements order by created_at desc) to '/tmp/measurements.csv' csv;"
-echo "Captured Time,Latitude,Longitude,Value,Unit,Location Name,Device ID,MD5Sum,Height,Surface,Radiation,Uploaded Time,Loader ID" | cat - /tmp/measurements.csv > /var/deploy/api.safecast.org/web_head/shared/system/measurements-out.csv
-cd /var/deploy/api.safecast.org/web_head/shared/system && tar -czf measurements-out.tar.gz measurements-out.csv
-mv measurements-out.tar.gz measurements.tar.gz && mv measurements-out.csv measurements.csv
-
-# rsync to the other server (this only runs on kangaroo)
-rsync -Havq --bwlimit 10000 /var/deploy/api.safecast.org/web_head/shared/system/measurements.* reindeer.api.safecast.org.c66.me:/var/deploy/api.safecast.org/web_head/shared/system/
-
-#
-# Now this is available as https://api.safecast.org/system/measurements.tar.gz
-#
+#!/bin/bash
+
+# 0: configuration -------------------------------------------------
+THIS_SERVER="kangaroo"	# confirm!
+OTHER_SERVERS="reindeer.api.safecast.org.c66.me"	# space separated list
+FINAL_DIR="/var/deploy/api.safecast.org/web_head/shared/system"
+FILENAME_STEM="measurements"
+
+DB_HOST="${POSTGRESQL_ADDRESS}"	# confirm, where is it set up in env?
+DB_USER="safecast"
+DB_DATABASE="safecast"
+DB_OUTFILE="/tmp/${FILENAME_STEM}.csv"
+# -----------------------------------------------------------------
+
+# 1: dump the data in csv format, no header
+psql --host "${DB_HOST}" --username "${DB_USER}" --dbname="${DB_DATABASE}" -c "\\copy (select captured_at, ST_Y(location::geometry), ST_X(location::geometry), value, unit, location_name, device_id, md5sum, height, surface, radiation, created_at, measurement_import_id from measurements order by created_at desc) to '${DB_OUTFILE}' csv;"
+
+# 2: add header and remove ${DB_OUTFILE}
+echo "Captured Time,Latitude,Longitude,Value,Unit,Location Name,Device ID,MD5Sum,Height,Surface,Radiation,Uploaded Time,Loader ID" | cat - ${DB_OUTFILE} > ${FINAL_DIR}/measurements-out.csv
```

Comment:
Guess we're not trying to change behavior here, but curious why we'd generate our own headers rather than embedding them in the select statement.

Reply:
Yes, exactly. I'll sync with the other scripts, then change the SQL to include column names.